machida/cpp/python-wallaroo.c

```diff
@@ -239,12 +239,64 @@ extern PyObject *initial_state(PyObject *computation)
   return pState;
 }
 
+extern PyObject *initial_accumulator(PyObject *aggregation)
+{
+  PyObject *pFunc, *pState;
+
+  pFunc = PyObject_GetAttrString(aggregation, "initial_accumulator");
+  pState = PyObject_CallFunctionObjArgs(pFunc, NULL);
+  Py_DECREF(pFunc);
+
+  return pState;
+}
+
+extern void aggregation_update(PyObject *aggregation, PyObject *data, PyObject *acc)
+{
+  PyObject *pFunc, *pState;
+
+  pFunc = PyObject_GetAttrString(aggregation, "update");
+  PyObject_CallFunctionObjArgs(pFunc, data, acc, NULL);
+  // PyObject_CallFunctionObjArgs(pFunc, aggregation, data, acc, NULL);
```

Comment:
Please remove zombie code comments in the final release of this

Reply:
addressed